src/libstd/process.rs

```diff
@@ -40,11 +40,15 @@ use thread;
 /// ```should_panic
 /// use std::process::Command;
 ///
-/// let output = Command::new("/bin/cat").arg("file.txt").output().unwrap_or_else(|e| {
```

Comment:
I think this example was targeting the `output()` convenience method, so perhaps it could continue to use that instead of `spawn()`?